Source/TCPServer.swift

```diff
@@ -26,18 +26,17 @@ import CLibvenice
 @_exported import IP
 
 public final class TCPServer: Host {
-    private let socket: tcpsock
+    private let socket: tcpsock?
 
-    public init(at host: String, on port: Int, queuing backlog: Int = 128, reusingPort reusePort: Bool = false) throws {
+    public init(at host: String, on port: Int, queuing backlog: Int = 128, reusePort: Bool = false) throws {
         let ip = try IP(localAddress: host, port: port)
         self.socket = tcplisten(ip.address, Int32(backlog), reusePort ? 1 : 0)
-        if nil == self.socket {
-            print("\n\nPort \(port) is already in use.\n\n")
-            throw TCPError.unknown(description: "Port \(port) is already in use.")
-        }
```

Comment:
the `try ensureLastOperationSucceeded()` will throw an appropriate error if `tcplisten` fails for some reason.